### ISSUE TITLE
Crash when scrolling with large library

### ISSUE THREAD

**Issue by Littlechay** [darktable-org/darktable#4749]
**Describe the bug**

I get repeated crashes when working in my library. I think often when scrolling to thumbnails that have not yet been built. Anyway I keep loosing my work on star ratings and such. Currently I have > 27,000 images 

**To Reproduce**

**Expected behavior**

I would expect to be able to scroll to any point in the library

**Screenshots**

**Platform (please complete the following information):**
 - Darktable Version:  3.0.1
 - OS: Windows 10
 -  yes 
 - Intel HD Graphics 620 (ASUStek Computer Inc) 21.20.16.4550

**Additional context**
[darktable_bt_QO8UI0.txt](https://github.com/darktable-org/darktable/files/4495301/darktable_bt_QO8UI0.txt)
[darktable_bt_H1IGJ0.txt](https://github.com/darktable-org/darktable/files/4495302/darktable_bt_H1IGJ0.txt)
[darktable_bt_1QM6I0.txt](https://github.com/darktable-org/darktable/files/4495303/darktable_bt_1QM6I0.txt)
[darktable_bt_NKXYI0.txt](https://github.com/darktable-org/darktable/files/4495304/darktable_bt_NKXYI0.txt)

**Comment by TurboGit** [comment]
I suppose the 3rd entry "yes" is for OpenCL. Just disbale it if you have an Intel card only and try again.

**Comment by Littlechay** [comment]
Hi

Yes the 'yes' was for OpenCL enabled and I realised when entering the report that probably should be off. Will see how it goes.

I also noted that I was getting crashes something to do with an unrecognised tag in TIFF files. I removed all the tiffs (and a bunch of left over files from other raw processors) and things are much better now. 

I also tried turning off the disk backend thumbnail cache. I'm not sure from the tooltip whether I should turn it on or off to help with browsing? 

Regards
Chris
On 18/04/2020 19:11:21, Pascal Obry <notifications@github.com> wrote:
I suppose the 3rd entry "yes" is for OpenCL. Just disbale it if you have an Intel card only and try again.
